sdk/communication/azure-communication-callingserver/CHANGELOG.md

```diff
@@ -0,0 +1,19 @@
+# Release History
+
+## 1.0.0-beta.1 (Unreleased)
+This is the first release of Azure Communication Calling Server Service. For more information, please see the [README][read_me].
+
+This is a Public Preview version, so breaking changes are possible in subsequent releases as we improve the product. To provide feedback, please submit an issue in our [Azure SDK for .NET GitHub repo](https://github.com/Azure/azure-sdk-for-net/issues).
+
+### Features Added
+- Create outbound call to an Azure Communication Service User or a phone number.
+- Hangup and delete the existing call.
+- Play audio in the call.
+- Outbound apis for Call Recording including start, pause, resume and stop.
+- Subscribe to tone and receive dtmf via events.
```

Comment:
DTMF

Reply:
Simplified to 'Subscribe to tone and receive tone selection via events.'